schema.go

```diff
@@ -9,21 +9,22 @@ type Fields map[string]Field
 // Wrap common schema fields from rest-layer
 var (
 	IDField = Field{
-		IDField,
+		Field:   schema.IDField,
+		NoIndex: false,
 	}
 
 	CreatedField = Field{
-		schema.CreatedField,
+		Field:   schema.CreatedField,
```

Comment:
I get a `mixture of field:value and value initializers` error without this change.